repos/keg-core/core/base/components/meta/versionDisplay.js

```diff
@@ -0,0 +1,75 @@
+import React, { useCallback, useMemo } from 'react'
+import { Button } from '@keg-hub/keg-components'
+import { Linking } from 'react-native'
+
+/**
+ * @param {string} homepage 
+ * @param {string} version 
+ * @return {Function} - callback for opening the release notes page for the 
+ * tap version
+ */
+const useOpenHomepageCallback = (homepage, version) => useCallback(
+  () => {
+    if (!homepage) return
+
+    const versionEndpoint = version ? `/v${version}` : ''
+    const uri = `${homepage}/releases/${versionEndpoint}`
+
+    Linking
+      .canOpenURL(uri)
+      .then(supported => supported 
+        ? Linking.openURL(uri)
+        : console.warn('Don\'t know how to open uri', uri)
+      )
+  },
+  [ homepage ]
+)
+
+/**
+ * Simple button that displays the current version of the tap and, if clicked,
+ * links the user to the release notes on github
+ * @param {Object} props
+ * @param {String} props.version - version to display (defaults to using the package.json version)
+ * @param {String} props.homepage - homepage to link to
+ * @param {Object} props.styles - custom styles for keg-components Button
+ * 
+ */
+export const VersionDisplay = props => {
+  const { 
+    version = process.env.TAP_VERSION || '<Missing Version>', 
+    homepage = process.env.TAP_HOMEPAGE,
+    styles
+  } = props
+
+  const openHomepage = useOpenHomepageCallback(homepage, version)
+
+  const buttonStyles = useMemo(() => ({ 
+    ...versionStyles, 
+    ...styles 
+  }), [ styles ])
+
+  return (
+    <Button
```

Comment:
You could have just used the `Link` component here. It has a number of benifits
  * It also uses the `Touchable` component just like the Button
  * You don't need to import the `Linking` library
  * You can just pass in the URL as an `href` prop
  * You can have it open in a new tab instead of the current one

That being said, you could also just
  * Make the content a `Text` component.
  * Then add `target="_blank"` and `href` props
  * That way you could keep the styles

Reply:
I actually forgot about the link component in keg-components. I'll just update it to use that.

Reply:
word

Reply:
updated to use Link 👍🏼